### ISSUE TITLE
BDD-X dataset download failed

### ISSUE THREAD

**Issue by miii-byte** [JinkyuKimUCB/explainable-deep-driving#10]
I encountered a 403 error when I tried to download the BDD-X dataset using wget. Can I get the BDD-X dataset in other ways?

**Comment by varungupta31** [comment]
Hey @miii-byte, were you able to resolve this issue and download the videos?

**Comment by ws-researcher** [comment]
@varungupta31 @miii-byte Do you have the data?

**Comment by varungupta31** [comment]
@ws-researcher No. However, I am not in requirement of it anymore, thanks.

**Comment by varungupta31** [comment]
@ws-researcher No, I reached out to the authors but failed to get a response. Also, [discussed around](https://github.com/valeoai/BEEF/issues/4) but I couldn't find it.

**Comment by roberto-amoroso** [comment]
@ws-researcher @varungupta31 @miii-byte Did you manage to solve the problem, or did you get an answer from the authors? The 403 error seems to persist...

**Comment by varungupta31** [comment]
@roberto-amoroso Nope, they won't respond.

**Comment by rishabh-akridata** [comment]
@varungupta31 Did you manage to download raw videos the BDD-X dataset?
Thanks.
